src/providers/ldap/ldap_child.c

```diff
@@ -61,7 +61,6 @@ static void sig_term_handler(int sig)
 static krb5_context krb5_error_ctx;
 #define LDAP_CHILD_DEBUG(level, error) KRB5_DEBUG(level, krb5_error_ctx, error)
 
-static const char *__ldap_child_krb5_error_msg;
```

Comment:
@lslebodn Thanks for comment.

Since I plan to remove `KRB5_SYSLOG`. And `KRB5_SYSLOG` is used only once in sssd code ie at line 530.
`static const char *__ldap_child_krb5_error_msg;
#define KRB5_SYSLOG(krb5_error) do { \
    __ldap_child_krb5_error_msg = sss_krb5_get_error_message(krb5_error_ctx, krb5_error); \
    sss_log(SSS_LOG_ERR, "%s", __ldap_child_krb5_error_msg); \
    sss_krb5_free_error_message(krb5_error_ctx, __ldap_child_krb5_error_msg); \
} while(0)`

Hence if we delete `KRB5_SYSLOG` macro will be left unused and Complier will throw warning of unused variable.
Or else we can place macro definition inside `#if 0  .... #endif`